### ISSUE TITLE
Missing headings

### ISSUE THREAD

**Issue by enewe101** [enewe101/digidemo#39]
A couple headings are missing and a misplaced "thank you message is being displayed"
- Add heading to the [registration page](http://luminocracy.org/userRegistration/) 
- Add a heading to the [reset password page](http://luminocracy.org/resetPassword/)
- Add a heading to the [feedback page](http://luminocracy.org/send_feedback/)
- Stop the feedback page from saying "thank you" when the form isn't correctly submitted.

**Comment by enewe101** [comment]
Hey @mlhuish, I've merged the change you made to overview.html into master.  You can now pull the newly merged master.  If, after pulling, you still have the branch `kill_petitions` locally (check this by running `git branch` and seeing if it is listed among the branches), then you can get rid of it by running `git branch -D kill_petitions`.

Issue #39 on our github issue tracker has similar things to be done.  In a few places, we have no header on the heading above a signup form, and we need one.  The task here would be to track down the appropriate template in `src/digidemo/templates/digidemo`, and then insert an <h2>Some heading</h2> at the appropriate place.

Until you get familiar with the templates, it's hard to figure out which file you need to edit to make that happen.  But you can figure it out in the following way.  Follow the link given in the issue description, and note the url.  Go into urls.py, and see which view function that url is mapped to.  Go into views.py, and find that view function.  Inside that view function, you'll find that the template being used is named somewhere, and there you go.

So, if you want to give issue #39 a shot, assign it to yourself and then, go for it!

**Comment by mlhuish** [comment]
Can do! I'll work on figuring this out today.

**Comment by mlhuish** [comment]
Hi @enewe101
I've been running into the following error when I click on the login or
register buttons

TemplateSyntaxError at /userRegistration/
Undefined variable or unknown value for: "GLOBALS.FEEDBACK_FORM"
...
any thoughts of how to solve this?
In the mean time I'll be attempting to set digidemo up on my other
computer.

**Comment by enewe101** [comment]
Yeah I recognize that.  I think that there were some changes made to the live server that didn't get pushed to master.

Could you try the following?  While on your own branch, commit any changes you have so far, then do:

    git pull
    git merge master

Now try navigating to the registration page.  Let me know if that fixes it!

**Comment by mlhuish** [comment]
I was still having issues on the original computer that we installed digidemo on, so I installed it on my other computer and everything is working great now! 
There were still errors durring the test, but they were ajax related and didn't affect the register and login pages. Great update to the installation guide. It's very easy to follow.

**Comment by mlhuish** [comment]
Also, let me know if you want the headings to be different

**Comment by enewe101** [comment]
Hey, @mlhuish , nice work!  Glad the new instructions are an improvement!

When I test your branch here, I get no errors.  So, maybe its something to do with the installation on your machines -- next time we're in the same room we can have a look together.

There's just one tweek for the feed-back form page.  If you look at the function `feedback()` inside `forms.py`, you'll see that there's a variable called `thank_you`, which is Boolean.  That variable gets passed to the template.  This means that, inside the template, you can include a [conditional tag](https://docs.djangoproject.com/en/1.6/ref/templates/builtins/#if), so that the "thank you message" only displays when the form got submitted properly.

**Comment by mlhuish** [comment]
OK! I understand. I didn't quite get what the issue was before. 
I've made the edit and will push my change.
Also, the form maintains the submitted data when the form is submitted correctly. Is this desireable or would you like the form to be cleared?

**Comment by enewe101** [comment]
Ok, this is great!  Nice work!

Just one last thing:
When the form is submitted properly, just say "Thank you!", _and prevent the form from even showing_.  You can do that by looking for the `include` tag responsible for showing the form, and put that tag inside the conditional.

And if there's an error, say "Oops!" or "Woops!" instead of "Error", and allow the form to show (exactly as it currently does).

**Comment by mlhuish** [comment]
Ahh ha! Of course! Cool. Thanks for the hand holding. Still leveling up my
skills.
